apollo-client.ts

```diff
@@ -3,13 +3,14 @@ import {
   InMemoryCache,
   NormalizedCacheObject,
 } from "@apollo/client";
+import { GRAPHQL_API } from "./config";
```

Comment:
We should probably move this into an environment variable, because then we could have it be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). But I didn't want to add that level of confusion in this PR.